src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,53 @@
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+
+  &:first-child {
+    margin-left: 0;
+  }
+}
+
+.nav__link {
+  position: relative;
+
+  display: flex;
+  align-items: center;
+
+  height: 60px;
+
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #060b35;
+
+  transition: color 0.3s;
```

Comment:
Ensure that the transition property is applied to specific properties rather than all properties. This is more performant and prevents unintended side effects. For example, use 'transition: color 0.3s;' instead of 'transition: all 0.3s;'.